meta/main.yml

```diff
@@ -24,44 +24,47 @@ galaxy_info:
         - bullseye
     - name: EL
       versions:
-        - 8
-        - 7
-        - 6
-    - name: Amazon Linux 2
+        - '7'
+        - '8'
+        - '9'
+    - name: Amazon Linux
       versions:
-        - any
+        - all
     - name: opensuse
       versions:
-        - 12.1
-        - 12.2
-        - 12.3
-        - 13.1
-        - 13.2
-        - 42.1
-        - 42.2
-        - 42.3
-        - 15.0
-        - 15.1
-        - 15.2
-        - 15.3
+        - '12.1'
+        - '12.2'
+        - '12.3'
+        - '13.1'
+        - '13.2'
+        - '42.1'
+        - '42.2'
+        - '42.3'
+        - '15.0'
+        - '15.1'
+        - '15.2'
+        - '15.3'
     - name: SLES
       versions:
         - 11SP3
         - 11SP4
-        - 12
+        - '12'
         - 12SP1
-        - 15
+        - '15'
     - name: Windows
       versions:
         - 2008x64
         - 2008R2
-        - 2012
+        - '2012'
         - 2012R2
-        - 2016
-        - 2019
-    - name: Darwin
-      versions:
-        - any
+        - '2016'
+        - '2019'
+    # Playbook works on Darwin platform, but we need to comment this out
+    # since schema used by ansible-lint to make validation do not reference Darwin:
+    # https://github.com/ansible/ansible-lint/blob/main/src/ansiblelint/schemas/meta.json
```

Comment:
Correct, but it does seem to have macos and macosx. These are probably more suitable here? (Not sure what's the difference, but you could probably do the research on that?)